tests/models/test_general.py

```diff
@@ -4,11 +4,14 @@
 
 import pytest
 from neomodel import UniqueIdProperty, One, RelationshipTo
+from datetime import datetime
```

Comment:
Since this is a standard library import, could you move this up to under `from __future__ import unicode_literals` please?